File: SuperiorPackGroup/BLL/ChangeObject.vb

Public Class ChangeObject
    Public Property PropertyName As String
    Public Property PrevValue As String
    Public Property NewValue As String
    Public Property ChangeID As Integer
    Public Property DateChanged As Date
    Public Property RecordID As Integer
End Class

Public Structure ChangeObjectLookUpKey
    Public ChangeID As Integer
    Public ProperyName As String
End Structure

File: SuperiorPackGroup/BLL/ChangeObject.cs
//INSTANT C# NOTE: Formerly VB project-level imports:
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;

namespace SuperiorPackGroup
{
	public class ChangeObject
	{
		public string PropertyName {get; set;}
		public string PrevValue {get; set;}
		public string NewValue {get; set;}
		public int ChangeID {get; set;}
		public DateTime DateChanged {get; set;}
		public int RecordID {get; set;}
	}

	public struct ChangeObjectLookUpKey
	{
		public int ChangeID;
		public string ProperyName;
	}
}